lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -373,39 +362,13 @@ constructUTxOIndex walletUTxO =
     walletUTxOIndex = UTxOIndex.fromMap $ toInternalUTxOMap walletUTxO
     cardanoUTxO = toCardanoUTxO Cardano.shelleyBasedEra walletUTxO
 
--- | Assumes all 'UTxO' entries have addresses with key payment credentials;
--- either normal, post-Shelley credentials, or boostrap/byron credentials
--- depending on the 'k' of the supplied 'TransactionLayer'.
-allKeyPaymentCredentials
-    :: forall k. TransactionLayer k 'CredFromKeyK SealedTx -> UTxOAssumptions
-allKeyPaymentCredentials tl = UTxOAssumptions
-    { txLayer = tl
-    , inputScriptLookup = Nothing
-    , inputScriptTemplate = Nothing
-    , txWitnessTag = transactionWitnessTag tl
-    }
-
--- | Assumes all 'UTxO' entries have addresses with script payment credentials,
--- where the scripts are both derived from the 'ScriptTemplate' and can be
--- looked up using the given function.
-allScriptPaymentCredentials
-    :: (W.Address -> (CA.Script KeyHash))
-    -> ScriptTemplate
-    -> TransactionLayer SharedKey 'CredFromScriptK SealedTx
-    -> UTxOAssumptions
-allScriptPaymentCredentials scriptLookup template tl = UTxOAssumptions
-    { txLayer = tl
-    , inputScriptLookup = Just scriptLookup
-    , inputScriptTemplate = Just template
-    , txWitnessTag = transactionWitnessTag tl
-    }
-
 balanceTransaction
-    :: forall era m changeState.
-        ( MonadRandom m
-        , IsRecentEra era
-        )
+    :: forall era m changeState
+     . ( MonadRandom m
+       , IsRecentEra era
+       )
```

Comment:
👌🏼 , I'll remove this and other formatting which place `.` on the new line because its not broken, it was caused originally by my misunderstanding of the current formatting convention. 

As for the "unrelated" aspect - I am still operating on the assumption that opportunistic formatting improvements (chores) are ok, as long as they are isolated in a separate commit.

Reply:
> 👌🏼 , I'll remove this and other formatting which place . on the new line because its not broken

Thanks! 🙇🏻 

>  it was caused originally by my misunderstanding of the current formatting convention.

No worries.

> As for the "unrelated" aspect - I am still operating on the assumption that opportunistic formatting improvements (chores) are ok, as long as they are isolated in a separate commit.

I think that formatting **_fixes_** are usually fine, provided that:

- they really are **_fixes_**: the original code violated the coding standards in some way.
- the changes are somehow related to the overall goal of the PR.

For example, if you're editing a function that violates the [line length limit](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md#limit-line-length-to-80-characters), I think it's totally acceptable to make a separate commit that adjusts the function to satisfy the limit.

OTOH, if the formatting changes are **_unrelated_** to the goal of the PR, or if they really are opportunistic, then I'd propose to put them in a separate PR, if possible.